Ptt/Common/BoardSearch/BoardSearchViewController.swift

```diff
@@ -121,7 +126,7 @@ extension BoardSearchViewController {
                     if let index = response.next_idx {
                         self?.startIdx = index
                     }
-                    self?.boards += response.list
+                    self?.boards = response.list
```

Comment:
Correct if I was wrong

Returned boards has a maximum number in each API response   
`+=` is for pagination purpose    
`=` seems like doesn't work if the search result more than the maximum number 

But since the default number is `200`, this is an edge case